docs/app/data/components-page.json

```diff
@@ -688,6 +688,24 @@
                     ]
                 },
                 {
+                    "title": "Item Display Panel - Inline",
+                    "component": "ComponentsItemDisplayPanelInlineComponent",
+                    "version": "Angular",
+                    "usage": [{
+                            "title": "Selector",
+                            "content": "ux-item-display-panel"
+                        },
+                        {
+                            "title": "Module",
+                            "content": "ItemDisplayPanelModule"
+                        },
+                        {
+                            "title": "Export as",
```

Comment:
Don't need exported as on a component, only for a directive so we can remove this